src/platform/packages/shared/kbn-alerts-ui-shared/src/alert_rule_from_vis_ui_action/types.ts

```diff
@@ -0,0 +1,15 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the "Elastic License
+ * 2.0", the "GNU Affero General Public License v3.0 only", and the "Server Side
+ * Public License v 1"; you may not use this file except in compliance with, at
+ * your election, the "Elastic License 2.0", the "GNU Affero General Public
+ * License v3.0 only", or the "Server Side Public License, v 1".
+ */
+
+export interface AlertRuleFromVisUIActionData {
```

Comment:
Lens folks, if there's a better package y'all can think of to export this from than `alerts-ui-shared` please advise. We need this data type in both x-pack and non-x-pack code, mostly in `chart_expressions` but also got references in `x-pack/lens` and `x-pack/triggers_actions_ui`.

I want to export it from a kbn package to avoid plugin dependency issues, I had to fight with those a lot to get this PR to build